### PR TITLE
doc: translate zh/tutorials/autoscaling-with-custom-metrics

### PR DESCRIPTION
**Pull request by B1F030**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
translate zh/tutorials/autoscaling-with-custom-metrics

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md [hunk 1]
```diff
@@ -1,42 +1,41 @@
...
 ---
-In Karmada, a FederatedHPA scales up/down the workload's replicas across multiple clusters, with the aim of automatically scaling the workload to match the demand.
+在 Karmada 中，为了自动扩展工作负载以满足需求，FederatedHPA 会跨多个集群扩/缩容工作负载。
 
-FederatedHPA not only supports resource metrics such as CPU and memory, but also supports custom metrics
-which may expand the use cases of FederatedHPA.
+FederatedHPA 不仅支持 CPU 和内存等资源指标，还支持自定义指标，这有助于扩展 FederatedHPA 的使用场景。
 
-This document walks you through an example of enabling FederatedHPA to automatically manage scale for a cross-cluster app with custom metrics.
+本文档将引导您完成一个启用 FederatedHPA 利用自定义指标来自动扩缩容跨集群应用的案例。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:8]:
```suggestion
本文档将引导您完成这样一个案例：启用 FederatedHPA 利用自定义指标来自动扩缩容跨集群应用。
```
make it more fluent

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md [hunk 2]
```diff
@@ -1,42 +1,41 @@
...
-The walkthrough example will do as follows:  
+演示案例将执行以下操作：
 
 ![federatedhpa-custom-metrics-demo](../resources/tutorials/custom_metrics.png)
 
-* One sample-deployment's pod exists in `member1` cluster.
-* The service is deployed in `member1` and `member2` cluster.
-* Request the multi-cluster service and trigger an increase in the pod's custom metrics(http_requests_total).
-* The replicas will be scaled up in `member1` and `member2` cluster.
+* `member1` 集群中存在一个示例 Deployment 下属的 Pod。
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:14]:
```suggestion
* `member1` 集群中存在一个示例 Deployment 的 Pod。
```
not understand 下属

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md [hunk 3]
```diff
@@ -203,9 +199,9 @@ NAME                 CLUSTER   TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)
 sample-app           member1   ClusterIP   10.11.29.250    <none>        80/TCP    3m53s   Y
 ```
 
-## Monitor your application in `member1` and `member2` cluster
+## 在 `member1` 和 `member2` 集群中监控您的应用
 
-In order to monitor your application, you'll need to set up a ServiceMonitor pointing at the application. Assuming you've set up your Prometheus instance to use ServiceMonitors with the app: sample-app label, create a ServiceMonitor to monitor the app's metrics via the service:
+为了监控您的应用，您需要配置一个指向该应用的 ServiceMonitor。假设您已配置 Prometheus 实例，使用带有 app:sample-app 标签的 ServiceMonitor，那么请创建一个 ServiceMonitor 以通过 Service 监控应用的指标：
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:204]:
```suggestion
为了监控您的应用，您需要配置一个指向该应用的 ServiceMonitor。假设您已配置 Prometheus 实例来使用带有 app:sample-app 标签的 ServiceMonitor，那么请创建一个 ServiceMonitor 以通过 Service 监控应用的指标：
```

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md [hunk 4]
```diff
@@ -389,45 +405,45 @@ As mentioned before, you need a multi-cluster service to route the requests to t
...
-## Install hey http load testing tool in member1 cluster
+## 在 member1 集群中安装 hey http 负载测试工具
 
-In order to do http requests, here you can use `hey`.
-* Download `hey` and copy it to kind cluster container.
-```
+为了发送 http 请求，这里我们使用 `hey`。
+* 下载 `hey` 并复制到 kind 集群容器中。
+```sh
 $ wget https://hey-release.s3.us-east-2.amazonaws.com/hey_linux_amd64
```
Comment (windsonsea) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:420]: 在复制粘贴命令后，用户还需要手动删除 `$` 符号，多了一步操作。所以很多纯命令的 block，没必要加 $ 吧

Comment (B1F030) [reply i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:420]: That makes sense, agree.

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md (hunk 1 above)
Comment (B1F030) [reply i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md]: Done.

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md (hunk 2 above)
Comment (B1F030) [reply i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md]: Fixed.

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md (hunk 4 above)
Comment (B1F030) [reply i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md]: 这个如果要改，其他文档应该也有很多处可以优化。之后可以在另外的 PR 中单独修改这一块。

**Comment by RainbowMango** [comment]
/assign @windsonsea @zhzhuang-zju 
to take another look

@B1F030 thanks for doing this, it'd be great if you could help [squash commits](https://karmada.io/docs/next/contributor/github-workflow/#squash-commits).

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md [hunk 5]
```diff
@@ -1,42 +1,41 @@
 ---
-title: FederatedHPA scales with custom metrics
+title: FederatedHPA 基于自定义指标扩容
```
Comment (zhzhuang-zju) [inline i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md:2]:
```suggestion
title: FederatedHPA 基于自定义指标扩缩容
```

**Review by zhzhuang-zju — COMMENTED** [review]
@B1F030 Thanks for your work.

others LGTM

File: i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md (hunk 5 above)
Comment (B1F030) [reply i18n/zh/docusaurus-plugin-content-docs/current/tutorials/autoscaling-with-custom-metrics.md]: Done.

**Review by windsonsea — COMMENTED** [review]
/lgtm

**Review by RainbowMango — APPROVED** [review]
/approve

**Pull request merged**